src/org/traccar/database/DeviceManager.java

```diff
@@ -387,4 +402,54 @@ public void sendCommand(Command command) throws Exception {
         }
         return result;
     }
+
+    public void updateMotionState(long deviceId, Position position) {
```

Comment:
It would be nice to extract logic back into motionhandler to avoid adding to much code device manager.